plugin/mlsdbonlinelocator.cpp

```diff
@@ -111,6 +115,25 @@ void MlsdbOnlineLocator::defaultVoiceModemChanged(const QString &modem)
     setupSimManager();
 }
 
+void MlsdbOnlineLocator::setWlanDataAllowed(bool allowed)
+{
+    if (m_wlanDataAllowed != allowed) {
+        m_wlanDataAllowed = allowed;
+    }
+    if (m_wlanDataAllowed && m_wlanServices.isEmpty() && m_networkManager) {
+        m_wlanServices = m_networkManager->getServices("wifi");
+    } else if (!m_wlanDataAllowed) {
+        m_wlanServices.clear();
+    }
+}
+
+void MlsdbOnlineLocator::setWlanSignalStrengthAllowed(bool allowed)
+{
+    if (m_wlanSignalStrengthAllowed != allowed) {
+        m_wlanSignalStrengthAllowed = allowed;
```

Comment:
While this is no QML API nowadays would expect symmetry i.e. if you have setter and getter also to provide nofity signal.

Reply:
done